packages/@jsii/integ-test/test/build-cdk.test.ts

```diff
@@ -0,0 +1,96 @@
+import { mkdtemp, remove } from 'fs-extra';
+import * as path from 'path';
+import * as Octokit from '@octokit/rest';
+import { downloadReleaseAsset, minutes, ProcessManager, extractFileStream } from '../utils';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+const JSII_DIR = path.dirname(require.resolve('jsii/package.json'));
+const JSII_PACMAK_DIR = path.dirname(require.resolve('jsii-pacmak/package.json'));
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN
+});
+
+describe('Build CDK', () => {
+  let buildDir: string;
+  let processes: ProcessManager;
+
+  beforeAll(async () => {
+    processes = new ProcessManager();
+    buildDir = await mkdtemp(path.join(__dirname, 'build'));
+  });
+
+  afterAll(async () => {
+    await processes.killAll();
+    await remove(buildDir);
+  });
+
+  test('can build latest cdk release', async () => {
+    // download latest release info
+    const release = await octokit.repos.getLatestRelease({
+      owner: 'aws',
+      repo: 'aws-cdk'
+    });
+
+    // download and extract code
+    const code = await downloadReleaseAsset(`https://api.github.com/repos/aws/aws-cdk/tarball/${release.data.tag_name}`);
+    const srcDir = path.join(buildDir, `aws-aws-cdk-${release.data.target_commitish.substring(0, 7)}`);
+    await extractFileStream(code, buildDir);
+
+    // install cdk dependencies
+    await processes.spawn('yarn', ['install'], {
+      cwd: srcDir
+    });
+
+    // build cdk build tools
+    await processes.spawn('npx', [
+      'lerna',
+      'run',
+      '--scope',
+      'cdk-build-tools',
+      '--scope',
+      'pkglint',
+      '--scope',
+      'awslint',
+      'build',
+    ], { cwd: srcDir });
```

Comment:
Maybe add `--stream`?

Also (and that is true for the other spawns, too), I kinda like logically grouping stuff, e.g:

```ts
    await processes.spawn('npx', [
        'lerna',
        '--stream',
        '--scope', 'cdk-build-tools',
        '--scope', 'pkglint',
        '--scope', 'awslint',
        'run', 'build',
      ], { cwd: srcDir });
```